src/components/AuctionStateHOC/index.tsx

```diff
@@ -199,6 +225,16 @@ export default (Component: React.ClassType<any, any, any>): React.ClassType<any,
       window.clearInterval(this.interval)
     }
 
+    claimSellerFunds = () => {
+      const { sell, buy, index, account, userCanClaim } = this.state
+      
+      console.log(
+        `claiming tokens for ${account} for
+        ${sell.symbol || sell.name || sell.address}->${buy.symbol || buy.name || buy.address}-${index}`,
+      )
+      return claimSellerFundsAndWitchdraw({ sell, buy }, index, userCanClaim, account)
```

Comment:
should be `claimSellerFundsAndWithdraw` (no 'c')

Reply:
No, no, its `witch-draw`, actually but in camelCase